bundle/config/resources.go

```diff
@@ -124,6 +124,18 @@ func (r *Resources) FindResourceByConfigKey(key string) (ConfigResource, error)
 		}
 	}
 
+	for k := range r.Dashboards {
+		if k == key {
+			found = append(found, r.Dashboards[k])
+		}
+	}
```

Comment:
This is from another PR, right? You can change the base branch to only show relevant differences there.

Reply:
yep, this is a re-base gone wrong